classes/db/DbMySQLi.php

```diff
@@ -409,7 +409,7 @@ public static function checkCreatePrivilege($server, $user, $pwd, $db, $prefix,
         }
 
         if ($engine === null) {
-            $engine = 'MyISAM';
+            $engine = 'InnoDB';
```

Comment:
should be better to adapt the code to use `getBestEngine` instead :)

Reply:
Sure, i added just the getBestEngine and works great. Thank you.